examples/cloudflare-sveltekit/README.md

```diff
@@ -0,0 +1,85 @@
+# Cloudflare SvelteKit Example
```

Comment:
I just realized this is in the example. Can we also add a guide that walks through a user setting up a sveltekit project with alchemy in alchemy-web/guides